client/src/templates/Challenges/classic/editor.css

```diff
@@ -33,6 +33,12 @@ textarea.inputarea {
   height: var(--breadcrumbs-height);
 }
 
+@media only screen and (max-width: 768px) {
```

Comment:
Just to confirm, you mentioned:

> When the viewport height is 768px or less

But this is listening to the width. Is that intentional?

Reply:
I meant width, sorry that was a mistake.